thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -438,6 +439,36 @@ public Response updateAnomalyFunction(@NotNull @QueryParam("id") Long id,
     return Response.ok(id).build();
   }
 
+  // Partially update anomaly function
+  @POST
+  @Path("/anomaly-function/update/properties")
+  public Response updateAnomalyFunctionProperties (@QueryParam("id") @NotNull Long id,
+      @QueryParam("config") @NotNull String propertiesJson) {
```

Comment:
can you align the query parameter naming and input naming here (may add java doc) to avoid the trouble of readability issue.